recipes/dataframe/all/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(cmake_wrapper)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+if (MSVC AND WIN32 AND BUILD_SHARED_LIBS)
```

Comment:
is it needed?
I see the code below:
```
if compiler == "Visual Studio" and self.options.shared:
            raise ConanInvalidConfiguration(
                "Could not support this specific configuration. Try static."
            )
```
so, either this line is useless, or code in conanfile?

Reply:
Update in the latest commit